src/SFA.DAS.ApplyService.Web/Views/Users/ExistingAccount.cshtml

```diff
@@ -15,63 +14,45 @@
 
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            @if (Model.ErrorMessages != null && Model.ErrorMessages.Count > 0)
-            {
-                <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
-                    <h2 class="govuk-error-summary__title" id="error-summary-title">
-                        There is a problem
-                    </h2>
-                    <div class="govuk-error-summary__body">
-                        <ul class="govuk-list govuk-error-summary__list">
-                            @foreach (var errorMessage in Model.ErrorMessages.DistinctBy(x => x.ErrorMessage))
-                            {
-                                <li><a href="#@errorMessage.Field">@errorMessage.ErrorMessage</a></li>
-                            }
-                        </ul>
-                    </div>
-                </div>
-            }
-
+         <partial name="_ValidationSummaryPartial" />
             <h1 class="govuk-heading-xl">Do you have an apprenticeship service (AS) sign in account?</h1>
 
-        </div>
-    </div>
+            <form method="post" asp-action="ConfirmExistingAccount" novalidate asp-controller="Users">
 
-
-    <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
-            <form method="post" asp-action="ConfirmExistingAccount" novalidate>
-
-                <div class="govuk-form-group @(showErrors ? "govuk-form-group--error" : "")" sfa-validation-for="FirstTimeSignin">
-                    @if (showErrors)
-                    {
-                        <span class="govuk-error-message">
-                            @Html.Raw(Model.ErrorMessages.FirstOrDefault(x => x.Field == "FirstTimeSignin")?.ErrorMessage)
-                        </span>
-                    }
+                <div class="govuk-form-group @(!ViewContext.ModelState.IsValid ? "govuk-form-group--error" : "")" sfa-validation-for="FirstTimeSignin" sfa-validationerror-class="form-group-error">
+               
+                    <span class="govuk-error-message">
+                        @Html.ValidationMessageFor(m => m.FirstTimeSignin, null, new { @class = "govuk-error-message error", id = "error-message-" + Html.IdFor(m => m.FirstTimeSignin) })
+                    </span>
```

Comment:
Should be: `<span class="govuk-error-message" asp-validation-for="FirstTimeSignin"></span>`